### DISCUSSION TITLE
I think, the highest recommended app version for patching should at least be 1 month old.

### DISCUSSION THREAD

**Discussion by Aryan-Chauhan** [anddea/revanced-patches#273] [Polls]
Because:

1. There are no differences in an app version which is 30 days old from today, so user will miss out on NOTHING. 

2. It will prevent unforseen bugs/unforseen crashing/unforseen incompatibality with patches/other undesirable behaviour, since 30 days old APKs are tried & tested.

3. Because of less bugs, there will be less bug reports.

**Comment by KrishnaSSH** [comment]
No, because kys

**Comment by anddea** [comment]
We cannot tell if there are bugs if no one tested it. So far, I cannot reproduce any issues that people mention. There's a possibility that newer versions will have more server side feature flags but most of the time it doesn't matter, feature flags are there for half a year before they get in prod.

Edit: code is always changing and sometimes we can clearly see that it might break something in the future, but sometimes it's not obvious. So, we can always drop support for some versions.
